app/src/main/java/org/sopt/sample/main/HomeListAdapter.kt

```diff
@@ -0,0 +1,48 @@
+package org.sopt.sample.main
+
+import android.content.Context
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView
+import org.sopt.sample.databinding.LayoutListItemBinding
+
+class HomeListAdapter(context: Context) : RecyclerView.Adapter<HomeListAdapter.HomeViewHolder>() {
+    private val inflater by lazy {
+        LayoutInflater.from(context)
+    }
+    private var itemList: List<HomeListItem> = emptyList()
+
+    class HomeViewHolder(
+        private val binding: LayoutListItemBinding
+    ) : RecyclerView.ViewHolder(binding.root) {
+
+        fun onBind(data: HomeListItem) {
+            binding.tvListItemName.text = data.name
+            binding.tvListItemBody.text = data.body
+            binding.ivMyFaceImg.setImageDrawable(binding.root.context.getDrawable(data.imageNo))
```

Comment:
```suggestion
            binding.ivMyFaceImg.setImageResource(data.imageNo)
```
`setImageResource`를 사용해보세요!!

Reply:
검색해봤는데  setImageDrawable는 Drawable 객체로 만들어진 이미지(수정 가능한 이미지 객체)를 설정
 할 때, setImageResource는 res 폴더에 있는 이미지를 설정할 때 이용한다고 하네요!!
ㅎ.. 너무 당연한 말일지도